lib/screens/e_commerce/order_list.dart

```diff
@@ -0,0 +1,48 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+import 'package:promdi_fe/screens/e_commerce/eWidgets/cardorder_list.dart';
+
+class OrderList extends StatefulWidget {
+  OrderList({Key? key}) : super(key: key);
+
+  @override
+  _OrderListState createState() => _OrderListState();
+}
+
+class _OrderListState extends State<OrderList> {
+  List data = [];
+
+  Future<String> loadJsonData() async {
+    var jsonText = await rootBundle.loadString('assets/json/order.json');
```

Comment:
Looks like when we finally start hitting the server, here is where we shall be getting back the actual order list. Perfect plan!!